OptimoveSDK/Sources/Classes/OptimoveConfig.swift

```diff
@@ -13,12 +13,16 @@ public struct Feature: OptionSet, @unchecked Sendable, CustomStringConvertible {
     public static let optimobile = Feature(rawValue: 1 << 1)
     /// Optimove feature.
     public static let optimove = Feature(rawValue: 1 << 2)
+    /// Preference center feature.
+    public static let preferenceCenter = Feature(rawValue: 1 << 4)
     static let delayedConfiguration = Feature(rawValue: 1 << 3)
 
     public init(rawValue: Int) {
         self.rawValue = rawValue
     }
 
+
+    //Question: Is this the way to go? I believe it's what Android sdk does
     public var description: String {
```

Comment:
What are you asking about?

If about the `description` method, then it is a required method from the `CustomStringConvertible` protocol. It creates a string representation of any object

Reply:
Ahh, sorry for being unclear. I was referring to the general approach on features